src/components/ProjectSettingsPage/index.jsx

```diff
@@ -88,54 +88,56 @@ class ProjectSettingsPage extends React.Component {
         params: { projectID },
       },
     } = this.props;
+    const trimprojectName = projectName.trim();
```

Comment:
```
    const trimprojectName = projectName.trim();
    const trimprojectDescription = projectDescription.trim()

```

@khalifan-kfan  A good way around this would be to use one function that returns a trimmed string. 
e.g. ` const trimmed = (input) => input.trim();`

Reply:
changed it to this in the new commit; 
const trimmed = (input) => input.trim();
    const trimprojectName = trimmed(projectName);
    const trimprojectDescription = trimmed(projectDescription);

let me know if its not what you ment